bindings/gumjs/runtime/console.js

```diff
@@ -15,6 +21,24 @@ class Console {
   error() {
     sendLogMessage('error', slice.call(arguments));
   }
+
+  count(label = 'default') {
+    if (!this.#counters.has(label)) {
+      this.#counters.set(label, 0);
+    }
+
+    const newValue = this.#counters.get(label) + 1;
+    this.#counters.set(label, newValue);
```

Comment:
This could be simplified to avoid two lookups per label. Perhaps something like:

```suggestion
    const newValue = (this.#counters.get(label) ?? 0) + 1;
    this.#counters.set(label, newValue);
```